src/components/Mentors.js

```diff
@@ -18,42 +18,27 @@ const MentorCard = ({ name, position, company, imageURL, active, phone, expertis
   </Col>
 );
 
+export const Mentors = () => {
+  const { data, isLoading } = useAirtableAPI('appexkZgUcQ9vucI9', 'mentors');
```

Comment:
maybe it'd be nice to have table ids as constants somewhere? unless theyre only going to be used once

Reply:
I think they should only be used once